options.go

```diff
@@ -145,6 +146,7 @@ func NewOptions() *Options {
 		ProxyWebSockets:     true,
 		HTTPAddress:         "127.0.0.1:4180",
 		HTTPSAddress:        ":443",
+		ProxyTimeOut:        30, //TIME_OUT in seconds
```

Comment:
Durations are in nanoseconds by default so you'll need to times this up to seconds
```suggestion
		ProxyTimeOut:        30 * time.Second, //TIME_OUT in seconds
```